render-ws-java-client/src/main/java/org/janelia/render/client/BackgroundCorrectionClient.java

```diff
@@ -0,0 +1,156 @@
+package org.janelia.render.client;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParametersDelegate;
+import ij.IJ;
+import ij.ImagePlus;
+import ij.process.ImageProcessor;
+import org.janelia.alignment.ImageAndMask;
+import org.janelia.alignment.filter.BackgroundCorrectionFilter;
+import org.janelia.alignment.filter.Filter;
+import org.janelia.alignment.spec.ChannelSpec;
+import org.janelia.alignment.spec.ResolvedTileSpecCollection;
+import org.janelia.alignment.spec.TileSpec;
+import org.janelia.alignment.util.ImageProcessorCache;
+import org.janelia.render.client.parameter.CommandLineParameters;
+import org.janelia.render.client.parameter.RenderWebServiceParameters;
+import org.janelia.render.client.parameter.ZRangeParameters;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+/**
+ * Create and store a background image that can be used to subtract a
+ * spatially varying background from a set of images.
+ * Originally developed for the CellMap jrc_mus_thymus_1 dataset.
+ *
+ * @author Michael Innerberger
+ */
+public class BackgroundCorrectionClient {
```

Comment:
With a deployed named filter configuration ( see [some examples here](http://renderer-dev.int.janelia.org:8080/render-ws/v1/namedFilterSpecLists) ) and `--filterListName ... --excludeAllTransforms` parameters, I think the [RenderTilesClient](https://github.com/saalfeldlab/render/blob/newsolver/render-ws-java-client/src/main/java/org/janelia/render/client/RenderTilesClient.java#L51-L59) would cover everything done by this client (along with some other features this client doesn't have).  I plan to use the RenderTilesClient for the thymus data set.  

The one nice thing this client has over the RenderTilesClient is that you don't need to deploy a named filter because you specify the filter parameters on the command line.  I'm waffling on whether we should:

a) keep this client and add a TODO to look at a good way to integrate this convenience into the RenderTileClient or 
b) simply remove this client as an almost-duplicate to reduce future maintenance.

Let me know what you think.

Reply:
I'd go for b. This client was just a means for me to test the whole process. If there is an existing client that can be used instead, we can definitely delete the new client.